src/Compiler/Facilities/LanguageFeatures.fs

```diff
@@ -102,6 +102,7 @@ type LanguageFeature =
     | UseBangBindingValueDiscard
     | BetterAnonymousRecordParsing
     | ScopedNowarn
+    | WarnOnInvalidDeclsInTypeDefinitions
```

Comment:
If changing the diagnostic to error, please also update the naming in places like here.